src/Mod/PartDesign/App/FeaturePad.cpp

```diff
@@ -98,10 +98,9 @@ App::DocumentObjectExecReturn *Pad::execute()
     Midplane.setReadOnly(hasReversed);
     Reversed.setReadOnly(hasMidplane);
 
-    Part::Feature* obj = 0;
     TopoDS_Shape sketchshape;
     try {
-        obj = getVerifiedObject();
+        Part::Feature* obj = getVerifiedObject();

```

Comment:
```suggestion
        getVerifiedObject();

```
Here you still have an unused 'obj' variable ;)

Reply:
Then I don't understand the compiler warning. The idea is to check if
`getVerifiedObject();`
fails. To do this,  must store the result of to a variable so the variable is used.

Do you know a way to silence the compiler?

Reply:
> I fear this can trigger warnings with other compilers.

Hmm, it seems to be OK also with the super strict MSVC settings I have. But OK, MSVC did not output the unused variable warning.

Reply:
No you don't need to store into a variable, hence my suggestion just to call the function without any assignment.
The idea here is to call 'getVerifiedObject' into a 'try' block because the function will throw an error if it finds a problem (that will then jump the code into the 'catch'. ;)